### PR TITLE
Feature to add organization to slack message

### PR DESCRIPTION
**Pull request by dlactin**
Added a feature to grab the organization from /etc/chef/client.rb and add it to slack message. Comes in handy if you are working with multiple chef organizations with similar node names/environments.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## 0.7.1
 - Version 0.7.0 broke most of the `message_detail_level` settings, only `basic` worked. 0.7.1 fixes it.
+- Added `send_organization` configuration which provides the optional ability to include the organization in /etc/chef/client.rb to Slack.
```
Comment (shortdudey123) [inline CHANGELOG.md]: `0.7.1` has already been released to this should go in an `Unreleased` section instead

Comment (dlactin) [inline CHANGELOG.md]: Moved to future 0.8.0 release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.8.0
```
Comment (shortdudey123) [inline CHANGELOG.md]: `unreleased`

**Comment by martinb3** [comment]
Hi there -- can you squash this into one commit and remove the changelog/metadata changes? Thanks!

**Comment by dlactin** [comment]
Opened another pull request with the requested changes. 

-- Edit: Updated this pull request instead, bad habit of closing and and creating new pull requests.

**Comment by shortdudey123** [comment]
Circle ruby version needs to be updated or `ruby_dep` locked to `~1.3`
@martinb3 thoughts on which one?

**Comment by martinb3** [comment]
Merged into master as of https://github.com/rackspace-cookbooks/chef-slack_handler/commit/a38fd8fa95238515e7c7357844cb523d3482a1fa. Thank you!

**Pull request closed without merging**